lib/auth/integration/integrationv1/awsoidc.go

```diff
@@ -417,6 +417,56 @@ func (s *AWSOIDCService) DeployDatabaseService(ctx context.Context, req *integra
 	}, nil
 }
 
+// DeployService deploys Services into Amazon ECS.
+func (s *AWSOIDCService) DeployService(ctx context.Context, req *integrationpb.DeployServiceRequest) (*integrationpb.DeployServiceResponse, error) {
```

Comment:
This probably needs a unit-test checking at least the auth part.

Reply:
Added two tests for this to ensure RBAC is applied